tools/dataset/preprocess_h36m.py

```diff
@@ -197,6 +253,8 @@ def _get_camera_params(self, camera, subject):
                                                     np.cos(z), 0], [0, 0, 1]])
         R = (R_x @ R_y @ R_z).T
         T = metadata_slice[3:6].reshape(-1, 1)
+        # convert unit from milimeter to meter
+        T *= 0.001
```

Comment:
Will this conflicts with annotations? Since the unit of 3D keypoints has already been converted to meter.

Reply:
It won't conflict with annotations. The unit of T needs to be the same as the unit of the 3D keypoint coordinates.